rollup.config.js

```diff
@@ -3,8 +3,12 @@ import bundleSize from 'rollup-plugin-bundle-size'
 import svelte from 'rollup-plugin-svelte'
 import resolve from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
-import { terser } from 'rollup-plugin-terser'
-import pkg from './package.json'
+import terser from '@rollup/plugin-terser'
+import { readFile } from 'fs/promises'
+
+const pkg = await readFile('./package.json', {
```

Comment:
I believe we can _still_ keep this as an `import` if we define the `exports` property on `package.json`, right?

E.g.:
```json
{
  "exports": {
    "./package.json": "./package.json"
  }
}
```

Aside from that, since we're moving to `"type": "module"` I also believe the `"exports"` key is actually required for a few build set-ups now? Setting this up shouldn't be that complicated since we're already doing both a `CommonJS` and `ES` builds (defined in the `main` and `module` fields respectively).

Reply:
Tested this in another project. You can still import it if you do:

```typescript
import pkg from './package.json' assert { type: 'json' };
```

Reply:
That's very useful - thank you! I've updated in the upcoming new version.